cocos/core/renderer/scene/model.ts

```diff
@@ -218,6 +227,8 @@ export class Model {
     protected _enabled = true;
     protected _visFlags = Layers.Enum.NONE;
 
+    protected _sortingPriority: number = SortingManager.getSortingPriority();
```

Comment:
getDefaultSortingPriority seems better

Reply:
No need to initialize with SortingManager, just make sure it will be updated